grails-app/services/org/pih/warehouse/inventory/ImportPickCommand.groovy

```diff
@@ -0,0 +1,30 @@
+package org.pih.warehouse.inventory
+
+import grails.validation.Validateable
+import org.apache.commons.lang.math.NumberUtils
+
+class ImportPickCommand implements Validateable {
+
+    String id
+    String code
+    String name
+    String lot
+    String expiration
```

Comment:
Let's be more specific here and use the built-in data binding if possible. Any reason not to?
```
String lotNumber
Date expirationDate
Location binLocation
```

Reply:
@jmiranda could you elaborate on  _"use the built-in data binding"_.
If you mean to use `bindData()` then it will be quite hard since I am doing the binding to the `ImportPickCommand` in the Service method where `bindData` is not available since it is on a controller method.